addons/l10n_ec/data/l10n_latam_document_type_data.xml

```diff
@@ -0,0 +1,404 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+	<data noupdate="0"> <!-- #TODO poner no update = 1 al completar la version 13 -->
```

Comment:
indentation, no need for a change of noupdate (we prefer the user not to update anything in the information of the document types)